src/main/java/org/apache/maven/doxia/cli/CLIManager.java

```diff
@@ -159,8 +164,12 @@ private static String getSupportedFormatAndEncoding()
 
     private static String getSupportedFormat()
     {
-        return "\nSupported Formats:\n from: " + join( DefaultConverter.SUPPORTED_FROM_FORMAT, ", " )
-            + " or autodetect" + "\n to:   " + join( DefaultConverter.SUPPORTED_TO_FORMAT, ", " )
+        String fromFormats = EnumSet.allOf( DefaultConverter.ParserFormat.class ).stream()
+                .map( f -> f.toString().toLowerCase() ).collect( Collectors.joining( ", " ) );
+        String toFormats = EnumSet.allOf( DefaultConverter.SinkFormat.class ).stream()
+                .map( f -> f.toString().toLowerCase() ).collect( Collectors.joining( ", " ) );
+        return "\nSupported Formats:\n from: " + fromFormats
+            + " or " + AUTO_FORMAT + "\n to:   " + toFormats
             + "\n";
```

Comment:
use system LS

Reply:
Done in https://github.com/apache/maven-doxia-converter/pull/38/commits/2b480deb54709437a7062846324d74bf3e415c37 (although not really related to this PR as `\n` was used everywhere else before)